src/server/csp.ts

```diff
@@ -121,6 +121,39 @@ const buildCsp = (
     media-src 'self' https://audio.guim.co.uk/
 `.trim();
 
+function cspEditions(
+	{ styles }: Assets,
+	thirdPartyEmbed: ThirdPartyEmbeds,
+): string {
+	return `
+	default-src 'self';
+	style-src 'self' ${assetHashes(styles)};
+	img-src 'self' https://static.theguardian.com https://*.guim.co.uk ${
+		thirdPartyEmbed.twitter
+			? 'https://platform.twitter.com https://syndication.twitter.com https://pbs.twimg.com data:'
+			: ''
+	};
+    script-src 'self' ${
+		thirdPartyEmbed.instagram ? 'http://www.instagram.com/embed.js' : ''
+	} https://interactive.guim.co.uk https://s16.tiktokcdn.com https://www.tiktok.com/embed.js https://sf16-scmcdn-sg.ibytedtos.com/ ${
+		thirdPartyEmbed.twitter
+			? 'https://platform.twitter.com https://cdn.syndication.twimg.com'
+			: ''
+	};
+    frame-src https://www.theguardian.com https://www.scribd.com ${
+		thirdPartyEmbed.instagram ? 'https://www.instagram.com' : ''
+	} https://www.facebook.com https://www.tiktok.com https://interactive.guim.co.uk ${
+		thirdPartyEmbed.spotify ? 'https://open.spotify.com' : ''
+	} ${
+		thirdPartyEmbed.youtube ? 'https://www.youtube-nocookie.com' : ''
+	} https://player.vimeo.com/ ${
+		thirdPartyEmbed.twitter
+			? 'https://platform.twitter.com https://syndication.twitter.com https://twitter.com'
+			: ''
```

Comment:
Same as above WRT Scribd, Instagram, TikTok, Facebook, Spotify, Vimeo and possibly YouTube (is YouTube required for Editions' video hosting?).

Reply:
We are dealing with standard articles in the first instance, are there no standard articles with Video embeds? If so looking at: https://miro.com/app/board/o9J_lcA5uzY=/ this fall under MainMedia?